sycl/test-e2e/LLVMIntrinsicLowering/bitreverse.cpp

```diff
@@ -0,0 +1,189 @@
+// Test that llvm.bitreverse is lowered correctly by llvm-spirv
+
+// RUN: %{build} -o %t.O2.out -O2
```

Comment:
Commit title should be '[SYCL][E2E] ..' instead of '[NFC] ...'

Reply:
Title changed.